templates/apps/sumo-aws-apps/common-org/common-org-management.yaml

```diff
@@ -403,119 +407,18 @@ Resources:
                 Action:
                   - lambda:ListTags
                   - lambda:GetFunctionConfiguration
+                  - lambda:UpdateFunctionConfiguration
                 Resource:
                   - !Sub arn:${AWS::Partition}:lambda:${AWS::Region}:${AWS::AccountId}:function:*
 
-              - Effect: Allow
-                Action:
```

Comment:
There were some new features EKS Protection in Amazon GuardDuty, https://docs.aws.amazon.com/guardduty/latest/ug/eks-runtime-monitoring-configuration.html
Can you please test it by choosing all the advanced options for ex EKS protection during CF deployment and verify whether stack creation is successfull .

Reply:
Tested EKS Protection stack creation is successfull. if we use new features Runtime Monitoring, we will change it

Reply:
stack creation is successful but are we getting findings from eks ? try creating a eks cluster and see if we get any findings. We cannot treat stack creation as truth log ingestion in sumo should be treated as successful testing  similar to how we missed config enable in network firewall in that case also stack creation was successful.